app/models/intake.rb

```diff
@@ -675,6 +671,23 @@ def new_record_token
     verifier.generate(SecureRandom.base36(24))
   end
 
+  # TODO: remove this after running the migration, need because hub client presenter
+  def needs_help_2018
```

Comment:
added this b/c was seeing errors like this https://codeforamerica.sentry.io/issues/6645673256/?alert_rule_id=1046555&alert_type=issue&notification_uuid=c781531e-a45e-46af-8f85-0f9793447e74&project=1880327&referrer=slack ; not 100% sure if this will solve the issue but also not sure how to test for it

Reply:
essentially between the deploy and the migration getting run, the hubclientpresenter is calling all the attributes on intake when it calls client.intake, by adding these empty methods we make sure it does die on intake.needs_help_2018; after the migration run on this PR we can remove these methods